<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>800</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #A09CAB;</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>801</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #484356;</string>
   </property>
   <widget class="QWidget" name="w_comp" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>411</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #F5F5F5;</string>
     </property>
     <property name="text">
      <string>NEEDED COMPETENCIES:</string>
     </property>
    </widget>
    <widget class="QWidget" name="w_comp_list" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>411</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QWidget" name="w_comp_item_0" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_0">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_0">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <zorder>l_name_0</zorder>
      <zorder>l_check_0</zorder>
     </widget>
     <widget class="QWidget" name="w_comp_item_1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_1">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_comp_item_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_comp_item_3" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_comp_item_4" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_comp_item_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="w_comp_item_6" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <widget class="QLabel" name="l_check_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="l_name_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>0</y>
         <width>381</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="w_status" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>0</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="l_status">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #f5f5f5;</string>
     </property>
     <property name="text">
      <string>Successful!</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_subtitle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>391</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #f5f5f5;</string>
     </property>
     <property name="text">
      <string>(Please swiping your card again to check out)</string>
     </property>
    </widget>
   </widget>
   <zorder>w_status</zorder>
   <zorder>w_comp</zorder>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>800</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #F4F3F6;</string>
   </property>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QLabel" name="l_contact">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #767184;</string>
     </property>
     <property name="text">
      <string>Support contact:</string>
     </property>
    </widget>
    <widget class="QLabel" name="l_phone">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #484356;</string>
     </property>
     <property name="text">
      <string>+ 47 926 94 666</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>0</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>static/digicheck.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="w_guide" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>800</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #625E6E;</string>
   </property>
   <widget class="QLabel" name="l_swiping">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>401</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;</string>
    </property>
    <property name="text">
     <string>Swiping your card before using the asset</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_username">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f5f5f5;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="l_timestamp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #f5f5f5;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>l_username</zorder>
   <zorder>l_timestamp</zorder>
   <zorder>l_swiping</zorder>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>497</x>
     <y>110</y>
     <width>258</width>
     <height>288</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>258</width>
      <height>258</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>static/machine.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #F5F5F5;</string>
    </property>
    <property name="text">
     <string>Asset-3</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>w_guide</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_4</zorder>
  <zorder>widget_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
